RecoTauTag/RecoTau/plugins/PATTauDiscriminationByMVAIsolationRun2.cc

```diff
@@ -110,6 +110,8 @@ namespace reco {
           mvaOpt_ = kDBoldDMwLTwGJ;
         else if (mvaOpt_string == "DBnewDMwLTwGJ")
           mvaOpt_ = kDBnewDMwLTwGJ;
+        else if (mvaOpt_string == "Phase2")
+          mvaOpt_ = kPhase2;
```

Comment:
as in other place, use something more descriptive than phase2 here and in other places

Reply:
kDBnewDMwLTwGJPhase2 will be used in its place.